packages/context/lit2/lit2-scope/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@lit-internal/context-lit2-scope",
```

Comment:
maybe this also needs `"type": "module"`. build seems to complain that files here are commonjs?